GPU/GPUCommon.cpp

```diff
@@ -570,8 +570,7 @@ void GPUCommon::DumpNextFrame() {
 }
 
 u32 GPUCommon::DrawSync(int mode) {
-	gpuStats.numDrawSyncs++;
```

Comment:
Hm, this is still a heavy sync and may impact timing, though I've not really seen this stat useful too often...

-[Unknown]

Reply:
Oops, I didn't actually mean to remove it here.

Reply:
Well the stat was actually added earlier in this PR, and just got lost in the revert of the history thingy. But still, I'll add it in, I like to have it.

Reply:
Oh, oops.  I was thinking I've never noticed it before but forgot it was added in this pull.  But yeah, I think it could be useful for looking at timing.

-[Unknown]